hs_core/hydroshare/hs_bagit.py

```diff
@@ -130,16 +125,43 @@ def create_bag(resource):
     to_file_name = os.path.join(resource.short_id, 'data', 'resourcemap.xml')
     istorage.saveFile(from_file_name, to_file_name, False)
```

Comment:
saveFile() runs an irods iput session which will raise SessionException if it fails.